recipe/meta.yaml

```diff
@@ -41,7 +41,7 @@ test:
     - test
   requires:
     - pytest
-    - shapely
+    - shapely  # [not osx and py=3.10]
```

Comment:
Not entirely sure what the syntax is for skipping if it is both osx and py=3.10.
See: https://github.com/conda-forge/shapely-feedstock/pull/91

Reply:
Just going to skip osx for now as it is for the tests.